core/templates/dev/head/pages/topic-editor-page/modal-templates/select-skill-modal.template.html

```diff
@@ -27,7 +27,7 @@ <h3 ng-if="skillSummaries.length > 0">
       <button class="btn btn-secondary" ng-click="cancelModal()"> Cancel </button>
       <button class="btn btn-success protractor-test-confirm-skill-button"
               ng-disabled="selectedSkillIds.length === 0"
-              ng-click="goToSelectDifficultyView()"> Next
+              ng-click="nextStep()"> Next
```

Comment:
Here and elsewhere, start function names with a verb.

Reply:
Done